web/components/FcDrawerViewData.vue

```diff
@@ -220,6 +248,7 @@ export default {
         school: null,
       },
       showFilters: false,
```

Comment:
for future work: rename _all_ of these old study filter pieces to `showFiltersStudy`, `filtersStudy`, etc. so that we can clearly distinguish / manage these.